### PR TITLE
[DOC] JSONAPIAdapter's findMany

### PR DESCRIPTION
**Pull request by sly7-7**

### CODE REVIEW

**Comment by locks** [comment]
Hi! Can you address the linting errors?

**Comment by sly7-7** [comment]
@locks Is it good enough now ?

**Comment by sly7-7** [comment]
@locks, I'm sorry, I don't understand the `@static`thing here, because this method is not a static one. 😕

**Comment by locks** [comment]
The Ember Data tests verify that the "table of contents" for API documentation did not change.
Since you are introducing `findMany` by documenting it, you need to tell https://github.com/emberjs/data/blob/master/packages/-ember-data/node-tests/fixtures/expected.js to expect `findMany`!

File: packages/adapter/addon/json-api.js [hunk 1]
```diff
@@ -218,9 +218,35 @@ const JSONAPIAdapter = RESTAdapter.extend({
...
+    ```js
+    {
+      "id": 1,
+      "title": "Rails is omakase",
+      "comments": [ 1, 2, 3 ]
+    }
+    ```
+    The IDs will be passed in this form:
+    ```
+    filter[id]=1,2,3
```
Comment (runspired) [inline packages/adapter/addon/json-api.js:236]: The example here appears to have very little to do with how this method operates and even less to do with JSON:API format. I'd suggest looking at the `findMany` documentation from the `MinimumAdapterInterface` for inspiration.

**Comment by sly7-7** [comment]
Sorry for the noise, I think I've been too much agressive during my housekeeping...

**Comment by locks** [comment]
Closing due to focus on the request manager going forward. Thanks for contributing!

**Pull request closed without merging**